sonar-text-plugin/src/test/java/org/sonar/plugins/common/DurationStatisticsTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * SonarQube Text Plugin
+ * Copyright (C) 2021-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.plugins.common;
+
+import java.nio.file.Paths;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.sonar.api.batch.sensor.internal.SensorContextTester;
+import org.sonar.api.testfixtures.log.LogTesterJUnit5;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class DurationStatisticsTest {
+  @RegisterExtension
+  LogTesterJUnit5 logTester = new LogTesterJUnit5();
+
+  @Test
+  void shouldRecordSimpleStatistics() {
+    var sensorContext = SensorContextTester.create(Paths.get("."));
+    sensorContext.settings().setProperty("sonar.text.duration.statistics", "true");
+    DurationStatistics durationStatistics = new DurationStatistics(sensorContext.config());
+    durationStatistics.timed("test-total", () -> {
+      try {
+        Thread.sleep(100);
+      } catch (InterruptedException e) {
+        throw new RuntimeException(e);
+      }
+      return null;
+    });
+    durationStatistics.log();
+
+    assertThat(durationStatistics.stats)
+      .hasSize(1);
+    assertThat(logTester.logs())
+      .anyMatch(s -> s.startsWith("Duration Statistics"))
+      .anyMatch(s -> s.contains("1 times"));
+  }
```

Comment:
I would like to see more tests:
- when one check is executed a few times (to verify the mean number) 
- when more checks are executed
I know that the output with be not deterministic because of `Thread.sleep(100)`, but I would like to see the whole format of the output.